orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/providers/aws/CaptureSourceServerGroupCapacityTask.groovy

```diff
@@ -40,28 +40,35 @@ class CaptureSourceServerGroupCapacityTask implements Task {
     def stageOutputs = [:]
     StageData stageData = stage.mapTo(StageData)
     if (stageData.useSourceCapacity) {
-      def sourceServerGroup = oortHelper.getTargetServerGroup(
-        stageData.source.account,
-        stageData.source.asgName,
-        stageData.source.region,
-        stageData.cloudProvider ?: stageData.providerType
-      ).orElse(null)
-
-      if (sourceServerGroup) {
-        // capture the source server group's capacity AND specify an explicit capacity to use when deploying the next
-        // server group (ie. no longer use source capacity)
+      if (!stageData.source && stageData.preferSourceCapacity) {
```

Comment:
I think there is no stageData.source when this is run in a pipeline, but I'm not sure we can reset the useSourceCapacity to false here because then clouddriver will not do the lookup based on the cluster name to find the previous server group